.github/workflows/release-e2e.yaml

```diff
@@ -0,0 +1,57 @@
+---
+name: release-e2e
+
+on:
+  release:
+    types: [created]
+  workflow_dispatch:
+
+jobs:
+  release-e2e-kind:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '~1.18'
+
+    - name: install_kind_binary
+      env:
+        KIND_VERSION: v0.12.0
+        KIND_OS: kind-linux-amd64
+      run: |
+        curl -sLo kind "https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/${KIND_OS}"
+        chmod +x kind
+        sudo mv kind /bin/
+
+    - name: start_kind_cluster
+      run: |
+        /bin/kind create cluster
+        /bin/kind export kubeconfig
+
+    - name: install_kubectl
+      run: |
+        curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+        sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+
+    - name: Set the release related variables
+      run: |
+          if [[ $GITHUB_REF == refs/tags/* ]]; then
+          echo IMAGE_TAG="${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+          fi
+
+    - name: release-install
+      run: |
+        kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.1/cert-manager.yaml
```

Comment:
Nit: The cert-manager version seems like another one that we'll forget to keep updated. Is there a way to glean this version from a centralized place in the repo?

Reply:
Yeah, is is already out of date with the latest release (1.8) so point well taken. I'll look into how to glean.

Reply:
@tylerslaton Reminded me that we have a Makefile variable for the cert-mgr version that I exported recently:

https://github.com/operator-framework/rukpak/blob/2d3120c7f7c97ab52b58c55f0161707b90566b72/Makefile#L16

I'm not sure how you can source a Makefile environment from a GHA environment outside of using Makefile targets though. Maybe here we could piggy back off the `make cert-mgr` target:

https://github.com/operator-framework/rukpak/blob/2d3120c7f7c97ab52b58c55f0161707b90566b72/Makefile#L109-L111

Which appears to emulate the same apply/wait commands?

Reply:
Well, that makes it easy. Will switch to piggybacking.